### DISCUSSION TITLE
Can't admin-merge if auto-merge feature is enabled

### DISCUSSION THREAD

**Discussion by Tratcher** [community/community#8797] [Pull Requests] [answered]
I have a PR I want to admin-merge to bypass some known CI issues. However, I'm only offered the auto-merge option, there's no way to directly merge.
![image](https://user-images.githubusercontent.com/1821173/145878154-68da1300-62d1-4311-8a8e-1960bb2f880f.png)

**Comment by mcastiello** [comment]
I'm having the same issue, it was fine up to this afternoon, then it suddenly stopped working

**Comment by johngill-bc** [comment]
Seeing the same issue on my PRs. Looking for resolution.

**Comment by clavat** [comment]
I guess it's a JS issue.

You can workaround this by inspecting the DOM and remove `hidden` attribute from the div with class `js-merge-box`.

**Comment by mcastiello** [comment, marked as answer]
It looks like it is working again, I was able to merge my PR this morning
